bots/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bots",
+  "name": "@foretold/bots",
```

Comment:
The bots should import the ``cdf`` package, which is needed. (Right now I believe it's manually copied & pasted, which is exactly what Lerna should help with)

Reply:
Fixed.